### ISSUE TITLE
Hang on LaTeX export when Fixed Width = 0

### ISSUE THREAD

**Issue by johnblommers** [vkbo/novelWriter#75]
When exporting a document to LaTeX and  the Fixed Width option is set to zero,
then novelWriter will wait at about the 12% point forever. 
Click cancel to recover gracefully.
The error message in the console is:

```
Traceback (most recent call last):
  File "/home/johnb/Applications/novelWriter/nw/gui/export.py", line 165, in _doExport
    outFile.addText(tHandle)
  File "/home/johnb/Applications/novelWriter/nw/convert/file/text.py", line 140, in addText
    self.theConv.doConvert()
  File "/home/johnb/Applications/novelWriter/nw/convert/text/tolatex.py", line 118, in doConvert
    self.theResult += "%s\n" % tWrap.fill(" ".join(thisPar))
UnboundLocalError: local variable 'tWrap' referenced before assignment
```

The workaround is to set the Fixed Width to something nonzero. 999 works pretty well as a workaround. However we believe the zero setting matters because we don't want to force line breaks early. LaTeX  typesetting does a better job.

**Comment by vkbo** [comment]
Ah, thanks for spotting this. It was the one thing I didn't remember to test. I am probably just missing a if > 0 check.

The wrapping for LaTeX is not for the final result, but to generate readable source.

**Comment by vkbo** [comment]
Fixed in PR #79.
